### DISCUSSION TITLE
A way to toggle/hide the bottom contextual help bar

### DISCUSSION THREAD

**Discussion by peppy** [jesseduffield/lazygit#1606] [Q&A] [answered]
I would like to hide the help bar at the bottom of the screen as it is of no use to me and taking up one vertical line of my screen. Searched through the config settings but couldn't find anything regarding this - is it already possible somehow?

**Comment by jesseduffield** [comment]
This is not currently possible but a config setting for this would make sense. Feel free to raise an issue if one does not already exist! Btw all that's required is for line 202 in pkg/gui/arrangement.go to be dynamic instead of hardcoded

**Comment by jesseduffield** [comment, marked as answer]
Update: this is now a thing as of the latest master branch via the gui.showBottomLine config key
